workflows/ts2fs.py

```diff
@@ -63,7 +63,7 @@ def generate_fs(ts, sample_sets, seq_len, neu_prop, nonneu_prop, output, format)
     nonneu_fs = mut_afs["non_neutral"]
 
     if format == 'dadi': _generate_dadi_fs(neu_fs, nonneu_fs, output)
-    elif format == 'polydfe': _generate_polydfe_fs(neu_fs, nonneu_fs, seq_len, neu_prop, nonneu_prop, output)
+    elif format == 'polydfe': _generate_polydfe_fs(neu_fs, nonneu_fs, seq_len, neu_prop, nonneu_prop, sample_sets, output)
```

Comment:
`sample_sets` is wrapped into a list, causing `len(sample_sets) == 1`

Reply:
get the `sample_size` before wrapping `sample_sets` into a list (line 61)

Reply:
so this won't work-- `allele_counts()` needs the list of `sample_sets`. i caught this bug on my version and fixed it however. can you please pull this branch again and see if it gives you the same error?

Reply:
Yes, it works after commenting out line 59 and 60, and changing line 45 to `freqs = allele_counts(ts, [sample_sets])`

Reply:
okay cool